cmd/antrea-agent/options.go

```diff
@@ -108,14 +109,38 @@ func (o *Options) validate(args []string) error {
 		if o.config.OVSDatapathType == ovsconfig.OVSDatapathNetdev {
 			return fmt.Errorf("exporting flows is not supported for OVS datapath type %s", o.config.OVSDatapathType)
 		} else {
+			// Check if it is TCP or UDP
```

Comment:
Maybe it deserves a separate func?

Reply:
Yes added another validation method in latest patch.